### PR TITLE
Language header follow up

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
Follow-up of https://github.com/owncloud/web/pull/8621

* Remove superfluous `useGraphClient` composable
* Bump `owncloud-sdk` to `3.1.0-alpha.4`

### CODE REVIEW

File: packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue [hunk 1]
```diff
@@ -134,7 +135,8 @@ export default defineComponent({
       }
 
       try {
-        await this.graphClient.users.getUser(this.user.onPremisesSamAccountName)
+        const client = this.clientService.graphAuthenticated
+        await client.users.getUser(this.user.onPremisesSamAccountName)
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue:139]: is the response irrelevant here? I know, was like that before, but still wondering now...

File: packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue [hunk 2]
```diff
@@ -271,7 +272,8 @@ export default defineComponent({
 
       if (this.user.onPremisesSamAccountName !== this.editUser.onPremisesSamAccountName) {
         try {
-          await this.graphClient.users.getUser(this.editUser.onPremisesSamAccountName)
+          const client = this.clientService.graphAuthenticated
+          await client.users.getUser(this.editUser.onPremisesSamAccountName)
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue:276]: Same here, response irrelevant? Why do we need the call then?

File: packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue (hunk 1 above)
Comment (JammingBen) [reply packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue:139]: Got me wondering as well, but now I remember: We request the user here to validate that the username does not yet exist. If the request succeeds, we throw a validation error. So the response itself is irrelevant.

Comment (kulmann) [reply packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue:139]: Could you add a comment in the code explaining this?

**Review by kulmann — APPROVED** [review]
💪

**Pull request merged**